src/abbrev.rs

```diff
@@ -650,10 +652,8 @@ pub mod tests {
         let expect = Some(Abbreviation::new(1,
                                             constants::DW_TAG_subprogram,
                                             constants::DW_CHILDREN_no,
-                                            vec![
-                    AttributeSpecification::new(constants::DW_AT_name,
-                                                constants::DW_FORM_string),
-                ]));
+                                            vec![AttributeSpecification::new(constants::DW_AT_name,
+                                                                    constants::DW_FORM_string)]));
```

Comment:
That's not great, but then again, rustfmt still doesn't deal with macros very well..